docs/best/react.md

```diff
@@ -217,7 +217,7 @@ autorun(() => {
 message.likes[2] = "Jennifer"
 ```
 
-This will react as expected. All array index assignments are detected, but only if `index <= length`.
+This will react as expected. All array index assignments are detected, but only if `index < length`.
```

Comment:
consult: 
https://github.com/mobxjs/mobx/blob/master/src/v5/types/observablearray.ts#L464

Reply:
This was correct, see https://github.com/mobxjs/mobx/blob/e9986766844c1da5cb337b8e88934ac0d4b81aa9/src/v5/types/observablearray.ts#L458, an assignment like `ar[ar.length] = x` will be interpreted as a `push`